BasicPlaybackSampleApp/BasicPlaybackSampleApp.xcodeproj/project.pbxproj

```diff
@@ -313,8 +313,7 @@
 				TargetAttributes = {
 					0B140F521A2E4709001324EE = {
 						CreatedOnToolsVersion = 6.1;
-						DevelopmentTeam = 556Y9XWD3N;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```

Comment:
Remove the changes you did in this file for the BasicPlaybackSampleApp

Reply:
Done